plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/AIConstants.java

```diff
@@ -41,5 +41,8 @@ public class AIConstants {
     public static final String AI_JOIN_RULE = "ai.joinRule";
     public static final int DEFAULT_CONTEXT_WINDOW_SIZE = 16384;
     public static final String AI_USE_STREAM_MODE = "ai.useStreamMode";
+
+    public static final String AI_FEATURE_CONNECTION_PROPERTY = "connection";
```

Comment:
I don't like these features. They are designed specifically for a particular web form, they don't have ny generic meaning.

Reply:
There should be only one abstraction on the model side:
1. fields required for connection
2. fields for model configuration

And this is not related in any way to the interface or forms, it's just a base meaning of how the engine can be configured, ui or other dependents might or might not use this separation for any reason, for example, to `connect` to the Engine first to load metadata (like available models)

If there is no such separation, this makes model UNUSABLE because you can't determine what data depends on what fields (available models depends on the connection status and connection fields)

**In simple words:
To load the model's information, you must first connect to the AI Model provider, which requires specifying specific fields. This is just how it works.**